CHANGELOG.asc

```diff
@@ -3,8 +3,8 @@
 Release Notes
 -------------
 
-Version 0.1.0 (TBD)
-~~~~~~~~~~~~~~~~~~~
+Version 0.1.0 (Release Date: April 11, 2017)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Can you please file an issue to cleanup the headers here to use the new headings format? Feel free to assign it to me.